packages/utils/upgrade/resources/codemods/5.0.0/sqlite3-to-better-sqlite3.json.ts

```diff
@@ -0,0 +1,29 @@
+import path from 'node:path';
+
+import type { modules } from '../../../dist';
+
+const transform: modules.runner.json.JSONTransform = (file, params) => {
+  const { cwd, json } = params;
+
+  const rootPackageJsonPath = path.join(cwd, 'package.json');
+
+  if (file.path !== rootPackageJsonPath) {
+    return file.json;
+  }
+
+  const j = json(file.json);
+
+  const targetProperties = ['sqlite3', 'vscode/sqlite3', 'sqlite-legacy'];
+
+  targetProperties.forEach((targetProperty) => {
+    const oldSqliteDependency = `dependencies.${targetProperty}`;
+    if (j.has(oldSqliteDependency)) {
+      j.remove(oldSqliteDependency);
+      j.set('dependencies.better-sqlite3', '9.0.0');
```

Comment:
Note for later: we should try to launch v5 with the latest version to avoid issues updating later on, but for now let's keep this what you have here since it matches the version we're using in the test apps and CI testing.

Reply:
Do you think it's worth putting a TODO?

Reply:
You don't want to replace it with latest since latest could in te future be a broken version or have breaking changes.
I think the best solution would be to use the use the version you use in the generator of that strapi version since that is the version the tests are ran agenst.
https://github.com/strapi/strapi/blob/762abbf7f8a8cae3cda9b6affcea50be8abc3ce9/packages/generators/app/src/utils/db-client-dependencies.ts#L7

Reply:
I didn't mean literally 'latest' I just mean a later version than 9.0.0 considering they're already on 9.2.2. And the generator will also be updated to match the version we use and test.